squirrel.worker/src/main/java/org/dice_research/squirrel/analyzer/impl/html/scraper/HtmlScraper.java

```diff
@@ -324,15 +328,12 @@ private String replaceCommands(String s) {
     	
     	List<Node> listNodes = new ArrayList<Node>();
     	
-    	@SuppressWarnings("unused")
-		boolean useResource = false;
-    	
     	 try {
           	
-          	if(cssQuery.startsWith("l(")) {
+          	if(cssQuery.startsWith("l")) {
           		
           		String val = cssQuery.substring(cssQuery.indexOf("(")+1,cssQuery.lastIndexOf(")"));
-//          		String label = uri.substring(uri.lastIndexOf("/")+1, uri.lastIndexOf("?"));
+          		String label = uri.substring(uri.lastIndexOf("/")+1, uri.length());
```

Comment:
Defining the label here is ok. However, why do you remove the search for `?` ?

Reply:
Removed this line as the label is being initialized in scrape method.